platform/platformsh.css

```diff
@@ -263,14 +264,23 @@ aside {
 
 .reveal code {
   font-family: var(--brand-font-mono), monospace;
+  background: rgb(48, 47, 69);
```

Comment:
This color has a very weird blue huge to it.  It's quite distracting.  Where'd you get it from?

Reply:
The prev color was from the UI code block, I updated it to match the website's code blocks but it's still blueish.